tests/test_fmu_dataio_cube.py

```diff
@@ -139,21 +138,20 @@ def test_cube_io_larger_case_ertrun(tmp_path):
 
     metadataout = out / ".volantis--what_descr.segy.yml"
 
+    assert metadataout.is_file() is True
 
-#    assert metadataout.is_file() is True
-
-# now read the metadata file and test some key entries:
-# with open(metadataout, "r") as mstream:
-#     meta = yaml.safe_load(mstream)
-# assert (
-#     meta["file"]["relative_path"]
-#     == "realization-0/iter-0/share/results/cubes/volantis--what_descr.segy"
-# )
-# assert meta["fmu"]["model"]["name"] == "ff"
-# assert meta["fmu"]["iteration"]["name"] == "iter-0"
-# assert meta["fmu"]["realization"]["name"] == "realization-0"
-# assert meta["data"]["stratigraphic"] is False
-# assert meta["data"]["bbox"]["xmin"] == 0.0
-# assert meta["data"]["bbox"]["xmax"] == 550.0
-
-# logger.info("\n%s", json.dumps(meta, indent=2))
+    # now read the metadata file and test some key entries:
+    with open(metadataout, "r") as mstream:
```

Comment:
pylint ~2.11 now complains about missing encoding here.

`meta = yaml.safe_load(Path(metadataout).read_text(encoding="utf8"))` is another option.